pkg/keto/cmd/create.go

```diff
@@ -75,6 +75,11 @@ func validateCreateFlags(c *cobra.Command, args []string) error {
 }
 
 func createClusterCmdFunc(c *cobra.Command, args []string) error {
+	cli, err := newCLI(c)
+	if err != nil {
+		return err
```

Comment:
There's a few of these 'err' check blocks in this file that could potentially benefit from some context written out in a log, unless it's returned to stdout further up the stack?